### PR TITLE
[new-app] Restructuring

### PR DESCRIPTION
**Pull request by binary-hamedanchi**

### CODE REVIEW

File: src/javascript/app_2/Modules/Statement/Components/card_list_desktop.jsx [hunk 1]
```diff
@@ -20,6 +21,7 @@ const CardListDesktop = ({
 export default connect(
     ({ modules }) => ({
         data            : modules.statement.data,
+
         columns         : [
             { title: localize('Daaaate'),          data_index: 'date' },
```
Comment (zakibinary) [inline src/javascript/app_2/Modules/Statement/Components/card_list_desktop.jsx]: Daaaate? 😄

Comment (binary-hamedanchi) [reply src/javascript/app_2/Modules/Statement/Components/card_list_desktop.jsx]: Good catch, though it's already fixed in the [next commit](https://github.com/binary-com/binary-static/pull/4060/commits/ece077072cbcc3c10f6bb35f145cfee4279ad84e#diff-8ce3a23410ba79a9192de9ebc6c85b75R30). 😄

File: src/javascript/app_2/Modules/Statement/Containers/statement.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            has_selected_date,
+            data,
+            is_loading,
+            is_mobile,
+        } = this.props;
+
+        return (
+            <React.Fragment>
+                <Filter />
+                { !is_mobile ?
```
Comment (negarn) [inline src/javascript/app_2/Modules/Statement/Containers/statement.jsx]: can we reverse the condition so we don't deal with `!`?

Comment (morteza-binary) [reply src/javascript/app_2/Modules/Statement/Containers/statement.jsx]: @negarn 
Sure, thanks.

File: src/javascript/app_2/Documents/STYLE_GUIDE.MD [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+* May contain both presentational and container components inside, 
+and usually, have some DOM markup and styles of their own.
+* Often allow containment via this.props.children.
+* Have no dependencies on the rest of the app, such as Flux actions or stores.
+* Don’t specify how the data is loaded or mutated.
+* Receive data and callbacks exclusively via props.
+* Rarely have their own state (when they do, it’s UI state rather than data).
+* Are written as functional components unless they need state, lifecycle hooks, 
+or performance optimizations.
+* Examples: Balance, AccountList, UserIfo, ...
```
Comment (negarn) [inline src/javascript/app_2/Documents/STYLE_GUIDE.MD]: missing `n` is `UserInfo`?

File: src/javascript/app_2/Documents/STYLE_GUIDE.MD [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+#### Do:
+
+* Keep all files immediately relevant to a component inside the given component directory.
+
+* Add a basic `index.js` to import the component and export it 
+to make it dead simple to import the component from elsewhere.
```
Comment (negarn) [inline src/javascript/app_2/Documents/STYLE_GUIDE.MD]: maybe other wording instead of `dead simple`? 😅

Comment (zakibinary) [reply src/javascript/app_2/Documents/STYLE_GUIDE.MD]: I like the use of `dead simple` though 😄

File: src/javascript/app_2/Documents/STYLE_GUIDE.MD [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+Containers/
+├── contract_type.jsx
+├── purchase.jsx
+├── trade.jsx
+└── trade_params.jsx
+```
+
+Containers should follow rules:
+
+* Are concerned with how thing work
```
Comment (negarn) [inline src/javascript/app_2/Documents/STYLE_GUIDE.MD]: `things`

File: src/javascript/app_2/Documents/STYLE_GUIDE.MD [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+
+## SERVICES
+
+All javascript files which are related to sending, fetching and handling WebSocket 
+requests and responses, are kept in `Services` directory.
+
+## STORES
+___The structure of this section is tentative.___
+
+This directory contains everything that directly related to MobX stores,
```
Comment (negarn) [inline src/javascript/app_2/Documents/STYLE_GUIDE.MD]: `everything that is`

File: src/javascript/app_2/Modules/Trading/Containers/contract_type.jsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import React          from 'react';
+import PropTypes      from 'prop-types';
+import ContractTypeWidget from '../Components/Form/ContractType/contract_type_widget.jsx';
+import { connect }    from '../../../Stores/connect';
```
Comment (negarn) [inline src/javascript/app_2/Modules/Trading/Containers/contract_type.jsx]: alignment

File: src/javascript/app_2/Documents/STYLE_GUIDE.MD (hunk 4 above)
Comment (ashkanx) [reply src/javascript/app_2/Documents/STYLE_GUIDE.MD]: I vote for dead simple 🗡

Comment (morteza-binary) [reply src/javascript/app_2/Documents/STYLE_GUIDE.MD]: @negar-binary 
What is your suggestion?

Comment (negarn) [reply src/javascript/app_2/Documents/STYLE_GUIDE.MD]: keep it if it's popular :)

**Pull request merged**